### DISCUSSION TITLE
Can't read content of useForm.errors

### DISCUSSION THREAD

**Discussion by IamNivelir** [react-hook-form/react-hook-form#2324] [Q&A]
I'm trying to check errors right after a value is set.
So you can imagine that there is a form with a manually registered field "title", on change will trigger this function:
```javascript
const handleTextChange = (e: ChangeEvent<HTMLInputElement>) => {
    const exists = getValues('title');
    !exists && register({ name: 'title' }, { required: true, maxLength: 25 });
    setValue('title', e.target.value, { shouldValidate: true });
    console.log('no errors: ', isEmpty(errors));
    console.log('errors: ', errors);
  };
```

And it registers and sets a new value just fine.
So while validation is passed I see that error is an empty object:

![image](https://user-images.githubusercontent.com/36738355/88218447-c0b56580-cc68-11ea-8371-1cdd83cf5ef1.png)

The problem comes when I try to read an error when the field would be empty or length exceeds 25 characters:

![image](https://user-images.githubusercontent.com/36738355/88218585-e7739c00-cc68-11ea-9a5b-daaff217107c.png)

I can't understand why it says that errors on this stage is an empty object, every possible check shows it (like this lodash/isEmpty). But when you will click to open an object in DevTools there will be values.

Would appreciate any help regarding this issue as I can't understand what am I doing wrong.

**Comment by bluebill1049** [comment]
could you share a simple codesandbox?
